src/Whoops/Handler/PlainTextHandler.php

```diff
@@ -211,6 +227,21 @@ private function getFrameArgsOutput(Frame $frame, $line)
     }
 
     /**
+     * Dump variable.
+     *
+     * @param mixed $var
+     * @return void
+     */
+    protected function dump($var)
+    {
+        if ($this->dumper) {
```

Comment:
It can be argued both ways. I prefer not having the check, as if the dumper has been set and it is now not callable, I would rather draw attention to it immediately, and not silently fallback on the other option, making people wonder why the set dumper does not activate.